site/docs/v1/tech/troubleshooting.adoc

```diff
@@ -157,6 +158,16 @@ include::docs/v1/tech/account-management/_account-management-troubleshooting.ado
 
 include::docs/v1/tech/shared/_troubleshooting-uninstall-upgrade-reinstall-rpm.adoc[]
 
+== Troubleshooting User Import
+
+If you are importing password hashes, ensure you provide only the hash in the `password` import field. For some formats you'll need to separate out the hash and the salt on the link:/docs/v1/tech/apis/users/#import-users[Import API]. If you do not, the import will succeed, but the user will not be able to login.
+
+For example, a bcrypt value of `$2a$10$N9qo8uLOickgx2ZMRZoMyeIjZAgcfl7p92ldGxad68LJZdL17lhWy` would be split out to the following import fields:
```

Comment:
We could add a link to some bcrypt doc that describes why we are splitting in this manner. This wiki page has a break down of the components of this string. https://en.wikipedia.org/wiki/Bcrypt

Reply:
Awesome, thanks, done.